test/molecule-role/molecule/default/tests/test_receiver.py

```diff
@@ -334,3 +348,21 @@ def _component_filtered(type_name, external_id_prefix, command):
         assert _component_filtered("process", "urn:process:/agent-centos", "/usr/bin/stress")
 
     util.wait_until(wait_for_components, 30, 3)
+
+
+def test_connection_network_namespaces_relations(host):
+    url = "http://localhost:7070/api/topic/sts_correlate_endpoints?limit=1000"
+
+    def wait_for_connection():
+        data = host.check_output("curl \"%s\"" % url)
+        json_data = json.loads(data)
+        with open("./topic-correlate-endpoint.json", 'w') as f:
+            json.dump(json_data, f, indent=4)
+
+        # assert that we find a outgoing localhost connection between 127.0.0.1 to 127.0.0.1 to port 9091 on
+        # agent-connection-namespaces host within the same network namespace.
+        outgoing_conn = _find_outgoing_connection_in_namespace(json_data, 9091, "agent-connection-namespaces", "127.0.0.1", "127.0.0.1")
```

Comment:
why we get the public ip for the connection being 127.0.0.1 instead of the internal docker ip ?
do we get the internal docker ip somehow ?